scipy/stats/_distn_infrastructure.py

```diff
@@ -1575,10 +1627,11 @@ def _ppf_to_solve(self, x, q, *args):
 
     def _ppf_single(self, q, *args):
         left = right = None
-        if self.a > -np.inf:
-            left = self.a
-        if self.b < np.inf:
-            right = self.b
+        _a, _b = self._get_support(*args)
+        if _a > -np.inf:
```

Comment:
Here if arguments are `array_like` the truthyness of the inequality is ambiguous, similar for `_b` just beneath.
For example, in case of `irwinhall` (or even `binom` support and `n` parameter is a 2 entry `array_like`.

Reply:
Interesting. Previously `self.a` and `self.b` were fixed numbers.  Now they are dependent on`*args`, which might not be specifying a single distribution, but a family of them.  That will also impact the determination of `left` and `right` in the block just below this.

Reply:
Commit 36a9e03 addressed this issue.

Reply:
>Commit 36a9e03 addressed this issue.

Yes.